css/styles_general.css

```diff
@@ -43,6 +45,39 @@ body {
 	font: 40px sans-serif, Arial;
 	text-align: center;
 	margin: 0;
+	padding: 10px 0;
+}
+
+.box_subtitle {
+	color: #31bebe;
+	font: 28px sans-serif, Arial;
+	text-align: left;
+	margin: 0;
+	padding: 20px 0
+}
+
+/* unvisited link */
+a:link {
+	text-decoration: none;
+	color: #ffffff;
```

Comment:
A link should never look the same as regular text.
https://www.nngroup.com/articles/guidelines-for-visualizing-links/